test/e2e/grpc_test.go

```diff
@@ -40,7 +40,7 @@ func unaryTest(t *testing.T, names test.ResourceNames, clients *test.Clients, ho
 	t.Logf("Connecting to grpc-ping using host %q and authority %q", host, domain)
 	conn, err := grpc.Dial(
 		host+":80",
-		grpc.WithAuthority(domain),
+		grpc.WithAuthority(domain+":80"),
```

Comment:
We can just remove this right?

Reply:
Not in the tests, because the host won't match the domain:

```
Connecting to grpc-ping using host "35.231.202.138" and authority "g-r-p-c-streaming-ping-ftgbeiff.serving-tests.example.com"
```